index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Chimezie's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+            <header class="head-container">
+                <div class="logo">
+                    <h3>Asouzu Chimezie</h3>
```

Comment:
1. It looks great, but it's highly recommended to put your logo in an anchor tag, so please ensure your logo is a link.
2. It's required to follow the exact Figma design, so please try using the same color text color for your logo.

![image](https://user-images.githubusercontent.com/34779928/198120859-d0b71ae3-c88f-444b-9783-bc12820c2538.png)